### ISSUE TITLE
Wiresless Transcievers behaving oddly.

### ISSUE THREAD

**Issue by Golrith** [zmaster587/AdvancedRocketry#1420]
### Version of Advanced Rocketry
AdvancedRocketry-1.12.2-1.4.1-107-universal

### Version of LibVulpes
LibVulpes-1.12.2-0.2.8-39-universal

### Version of Minecraft
12.2.2

### ##Description of the problem
The wireless transcievers are behaving oddly.

Trying to connect up to the Astrobody Data Processor.
The mass data does not transmit
The composition data stored in satellite terminal is transmitted, but anynew data vanishes as it tried to transmit
Distance data transmits fine.

There also appears to be a specific steps to make them work at all, with placing, linking, setting mode and turning on/off. Get this process wrong, and they do not work at all.

I tried using the obsolete data cables, and they did not want to send data at all.

**Comment by Hachuurui** [comment]
I was noticing this exact same behavior.  Thought I was losing my mind for a second.

**Comment by zmaster587** [comment]
Would you be able to explain the steps required to reproduce the problem?  What order do you have to do things in to reliably replicate the issue?

**Comment by Hachuurui** [comment]
In my case, I attached the receivers to both satellite terminal and astro data processor, link them using a linker, set satellite term to extract then turn them on. Its odd cause they all worked fine for a while. Then suddenly went all wonky after a couple days. 


I can say that this has been happening on a multiplayer server using the enigmatica 2 expert pack. (for myself) and I'm using ender io conduits and a rftools powercell to power them.

Conduits underneath, transmitters on the backs of the satellite terminals and on the 3 data blocks for the astro data processor.


I did notice that they seem to keep a "location" memory. I was going to see if it fixed itself by moving the setup to another chunk and try hooking it all up again. I'll be able to test this theory later this afternoon.

**Comment by zmaster587** [comment]
Alright, I was wondering if maybe it's some kind of chunk loading order thing.

**Comment by Hachuurui** [comment]
I was thinking the same actually. so I broke all of them and waited for reset. then after replacing and setting. they all worked fine. After reset again, everything is broken again. There is an IC2 Chunkloader in the area. so this could be the issue. I also noticed that other machines such as the observatory and electrolyzer also have weird behavior after reset. such as all meteors having only air one reset then full of ore next reset. 

I'll remove the chunk loader and see how the behavior is after resets.

**Comment by Hachuurui** [comment]
Removing the chunkloaders do seem to have fixed the issue with satellites terminals and the transceivers. I'm waiting for night to see of observatory is also fixed.

**Comment by Hachuurui** [comment]
It did not fix the observatory. so that may be a different issue.

**Comment by MicahHaack** [comment]
I can confirm this is still an issue. After messing around a bit, I found that only the distance data mode transfers properly.  Both mass and composition seem to disappear into the void.

I double checked that the satellites are making data with the interface by observing it while not attached to the wireless transceiver.

Next, if there was a buffer in the interface, I then performed these steps in order:
1. Attach a transceiver on both sides.
2. Link them with the linker.
3. Set the one attached to the interface to extract.
3. Enable the one attached to the data output.
4. Enable the one attached to the interface.

When done in this order, distance will transfer properly over time.
However, both mass and composition will immediately transfer the current buffer over, and then any data afterwards disappears.  It does not get stored in the output or the interface.

Connecting the transceiver in any other way will cause it to not work for any data, even distance.

Hope this helps.
-Kel
